### ISSUE TITLE
Issue with ppv help

### ISSUE THREAD

**Issue by deleenm** [jcbird/ppv#3]
The help on ppv, seems to be broken.

help(ppv)

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-29-599ad57427c3> in <module>
----> 1 help(ppv)

~/anaconda3/envs/ppv/lib/python3.8/_sitebuiltins.py in __call__(self, *args, **kwds)
    101     def __call__(self, *args, **kwds):
    102         import pydoc
--> 103         return pydoc.help(*args, **kwds)

~/anaconda3/envs/ppv/lib/python3.8/pydoc.py in __call__(self, request)
   1906     def __call__(self, request=_GoInteractive):
   1907         if request is not self._GoInteractive:
-> 1908             self.help(request)
   1909         else:
   1910             self.intro()

~/anaconda3/envs/ppv/lib/python3.8/pydoc.py in help(self, request)
   1965             else: doc(str, 'Help on %s:', output=self._output)
   1966         elif isinstance(request, Helper): self()
-> 1967         else: doc(request, 'Help on %s:', output=self._output)
   1968         self.output.write('\n')
   1969 

~/anaconda3/envs/ppv/lib/python3.8/pydoc.py in doc(thing, title, forceload, output)
   1685     try:
   1686         if output is None:
-> 1687             pager(render_doc(thing, title, forceload))
   1688         else:
   1689             output.write(render_doc(thing, title, forceload, plaintext))

~/anaconda3/envs/ppv/lib/python3.8/pydoc.py in render_doc(thing, title, forceload, renderer)
   1678         object = type(object)
   1679         desc += ' object'
-> 1680     return title % desc + '\n\n' + renderer.document(object, name)
   1681 
   1682 def doc(thing, title='Python Library Documentation: %s', forceload=0,

~/anaconda3/envs/ppv/lib/python3.8/pydoc.py in document(self, object, name, *args)
    378         # by lacking a __name__ attribute) and an instance.
    379         try:
--> 380             if inspect.ismodule(object): return self.docmodule(*args)
    381             if inspect.isclass(object): return self.docclass(*args)
    382             if inspect.isroutine(object): return self.docroutine(*args)

~/anaconda3/envs/ppv/lib/python3.8/pydoc.py in docmodule(self, object, name, mod)
   1188             contents = []
   1189             for key, value in data:
-> 1190                 contents.append(self.docother(value, key, name, maxlen=70))
   1191             result = result + self.section('DATA', '\n'.join(contents))
   1192 

~/anaconda3/envs/ppv/lib/python3.8/pydoc.py in docother(self, object, name, mod, parent, maxlen, doc)
   1443     def docother(self, object, name=None, mod=None, parent=None, maxlen=None, doc=None):
   1444         """Produce text documentation for a data object."""
-> 1445         repr = self.repr(object)
   1446         if maxlen:
   1447             line = (name and name + ' = ' or '') + repr

~/anaconda3/envs/ppv/lib/python3.8/reprlib.py in repr(self, x)
     50 
     51     def repr(self, x):
---> 52         return self.repr1(x, self.maxlevel)
     53 
     54     def repr1(self, x, level):

~/anaconda3/envs/ppv/lib/python3.8/pydoc.py in repr1(self, x, level)
   1047             if hasattr(self, methodname):
   1048                 return getattr(self, methodname)(x, level)
-> 1049         return cram(stripid(repr(x)), self.maxother)
   1050 
   1051     def repr_string(self, x, level):

~/anaconda3/envs/ppv/lib/python3.8/site-packages/ppv-0.0.1-py3.8.egg/ppv/__init__.py in __repr__(self)
     20     def __repr__(self):
     21         first = f'Information container for ppv'
---> 22         second = f'Available properties: {*self._available_commands,}'
     23         return f'{first}\n{second}'
     24 

TypeError: 'method' object is not iterable

**Comment by jcbird** [comment]
Yep, thanks again! I was trying to be too fancy with the __repr__ method. Fixed in b494d5083b52d8533df6cb115ef5a2ec28061edd .
Let me know when you get a chance.
